flatdata-py/generator/tests/generators/test_rust_generator.py

```diff
@@ -5,6 +5,16 @@
 
 from generator.generators.RustGenerator import RustGenerator
 from .assertions import *
+from nose.tools import eq_
+
+
+def test_format_numeric_literals():
+    eq_(RustGenerator._format_numeric_literal(1), "1")
+    eq_(RustGenerator._format_numeric_literal(123), "123")
+    eq_(RustGenerator._format_numeric_literal(1000000), "1_000_000")
```

Comment:
What do you think about adding a manual test for 1 to 10 zeroes?

Reply:
done, without 10 zeroes, but more.